servers/rendering/renderer_rd/shaders/skeleton.glsl

```diff
@@ -100,7 +100,7 @@ void main() {
 
 		for (uint i = 0; i < params.blend_shape_count; i++) {
 			float w = blend_shape_weights.data[i];
-			if (w > 0.0001) {
+			if ((w < 0.0001) || (w > 0.0001)) {
```

Comment:
Isn't this an and? I think it's trying to exclude 0.0f, please check.

Reply:
Yup :facepalm: 

Will amend shortly

Reply:
Wait, this seems fine to me.

If the weight is not zero (i.e. less than *or* greater the epsilon).

For it to be an *and* operation, it would need for the weight to be simultaneously less than and greater than the epsilon, which it could not be?